Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM golang:1.20-alpine3.18 as builder
+FROM --platform=$BUILDPLATFORM golang:1.20-alpine3.18 as builder
 WORKDIR $GOPATH/src/go.k6.io/k6
-COPY . .
+ARG TARGETOS TARGETARCH
 RUN apk --no-cache add git=~2
-RUN CGO_ENABLED=0 go install -a -trimpath -ldflags "-s -w -X go.k6.io/k6/lib/consts.VersionDetails=$(date -u +"%FT%T%z")/$(git describe --tags --always --long --dirty)"
+RUN --mount=target=.,readonly \
```

Comment:
On systems with selinux (fedora, redhat, other redhat derivatives :P), you need `,Z` after this.

I guess this makes it way faster then `COPY` from my test? But I am not certain if the `,Z` works on not selinux, so it might be better if we keep `COPY` if it doesn't.

Reply:
Do you get the same error if you build via `buildx`?

Reply:
>The --mount flag does not support z or Z options for modifying selinux labels.

Also, set `Z` doesn't seem to be a pattern to encourage so considering this is not really a requirement and the negative impact it could have I'm going to roll back this change.

Reply:
@mstoykov I expect you have the same issue with the go build cache folder. Right?

Reply:
I removed both .. to be honest - but it shouldn't matter in that case as the problem is that I am mounting filesystem that is selinux protected and don't do anything to make it work. While I think the cache is just a container volume. 

Testing it - it didn't error out so it worked :+1: 

No idea how useful it is to add it though. Most people will not be building this constantly and k6 compilation times are already pretty okay. I also expect this will mostly be run by github actions. And I will be plesently surprised if this helps there at all if we constantly bounce between runners.